helm/hpcc/values.yaml

```diff
@@ -144,14 +147,21 @@ eclagent:
   useChildProcesses: false
   ## type may be 'hthor' (the default) or 'roxie', to specify that the roxie engine rather than the hthor engine should be used for eclagent workunit processing
   type: hthor
+  ## The following resources apply to child hThor pods when useChildProcesses=false, otherwise they apply to hThor pod.
+  #resources:
+  #  cpu: "1"
+  #  memory: "1G"
 
 - name: roxie-workunit
   replicas: 1
   prefix: roxie_workunit
   maxActive: 20
   useChildProcesses: true
   type: roxie
-    
+  #resources:
+  #  cpu: "1"
+  #  memory: "1014M"
```

Comment:
odd amount?

Reply:
This left over from testing.  I wanted to make sure the values were coming from the right place.